### PR TITLE
Management page

### PR DESCRIPTION
**Pull request by walfly**

### CODE REVIEW

File: packages/components/src/StepProcess/StepProcess.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+import { StepProcess, StepProps } from "./StepProcess";
+import { StepHeader } from "./StepHeader";
+import { StepStyled } from "./StepStyled";
+
+const Step1 = (props: StepProps): JSX.Element => {
+  console.log(props);
```
Comment (dankins) [inline packages/components/src/StepProcess/StepProcess.stories.tsx]: should this be removed?

File: packages/core/src/contracts/multisig/multisig.ts [hunk 2]
```diff
@@ -66,9 +66,16 @@ export class Multisig extends BaseWrapper<MultiSigWalletContract> {
     await this.requireOwner();
 
     const options = await this.instance.addOwner.getRaw(owner, { gas: 0 });
+    console.log(options);
```
Comment (dankins) [inline packages/core/src/contracts/multisig/multisig.ts]: extraneous logging statement

File: packages/components/src/Collapsable.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  public componentDidMount(): void {
+    this.collapseArea!.addEventListener("transitionend", () => {
+      if (this.state.open) {
+        this.setState({ height: null });
+      }
+    });
+  }
+
+  public componentWillReceiveProps(nextProps: CollapsableProps): void {
+    if (nextProps.open !== this.props.open) {
```
[the remark this answers is not in the document]
Comment (dankins) [reply packages/components/src/Collapsable.tsx]: nvm, googled it:
https://stackoverflow.com/questions/42273853/in-typescript-what-is-the-exclamation-mark-bang-operator-when-dereferenci

File: packages/components/src/DetailTransactionButton.tsx [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+  DetailTransactionButtonState
+> {
+  constructor(props: DetailTransactionButtonProps) {
+    super(props);
+    this.state = {
+      price: 0,
+      priceFailed: false,
+      isProgressModalVisible: false,
+    };
+    this.devinePrice = debounce(this.devinePrice.bind(this), 1000);
```
Comment (tobek) [inline packages/components/src/DetailTransactionButton.tsx]: `divine`!

**Pull request merged**